plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestExasolTypeMapping.java

```diff
@@ -0,0 +1,301 @@
+    }
+
+    @Test
+    void testTimestamp()
+    {
+        testTimestamp(UTC);
+        testTimestamp(jvmZone);
+        // using two non-JVM zones so that we don't need to worry what Exasol system zone is
+        testTimestamp(vilnius);
+        testTimestamp(kathmandu);
+        testTimestamp(TestingSession.DEFAULT_TIME_ZONE_KEY.getZoneId());
+    }
+
+    private void testTimestamp(ZoneId sessionZone)
+    {
+        Session session = Session.builder(getSession())
+                .setTimeZoneKey(TimeZoneKey.getTimeZoneKey(sessionZone.getId()))
+                .build();
+
+        SqlDataTypeTest.create()
+                .addRoundTrip("timestamp", "NULL", createTimestampType(3), "CAST(NULL AS TIMESTAMP)")
+                .addRoundTrip("timestamp", "TIMESTAMP '0001-01-01 00:00:00.000'", createTimestampType(3), "TIMESTAMP '0001-01-01 00:00:00.000'") // min value in Exasol
+                .addRoundTrip("timestamp", "TIMESTAMP '1582-09-30 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1582-09-30 13:18:03.123'")
+                .addRoundTrip("timestamp", "TIMESTAMP '1582-10-01 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1582-10-01 13:18:03.123'")
+                .addRoundTrip("timestamp", "TIMESTAMP '1582-10-02 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1582-10-02 13:18:03.123'")
+                .addRoundTrip("timestamp", "TIMESTAMP '1582-10-14 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1582-10-15 13:18:03.123'")
+                .addRoundTrip("timestamp", "TIMESTAMP '1582-10-15 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1582-10-15 13:18:03.123'")
+                .addRoundTrip("timestamp", "TIMESTAMP '1958-01-01 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1958-01-01 13:18:03.123'")
+                .addRoundTrip("timestamp", "TIMESTAMP '2019-03-18 10:01:17.987'", createTimestampType(3), "TIMESTAMP '2019-03-18 10:01:17.987'")
+                .addRoundTrip("timestamp", "TIMESTAMP '2018-10-28 01:33:17.456'", createTimestampType(3), "TIMESTAMP '2018-10-28 01:33:17.456'")
+                .addRoundTrip("timestamp", "TIMESTAMP '2018-10-28 03:33:33.333'", createTimestampType(3), "TIMESTAMP '2018-10-28 03:33:33.333'")
+                .addRoundTrip("timestamp", "TIMESTAMP '2018-03-25 03:17:17.000'", createTimestampType(3), "TIMESTAMP '2018-03-25 03:17:17.000'")
+                .addRoundTrip("timestamp", "TIMESTAMP '1986-01-01 00:13:07.000'", createTimestampType(3), "TIMESTAMP '1986-01-01 00:13:07.000'")
+                .addRoundTrip("timestamp", "TIMESTAMP '9999-12-31 23:59:59.999'", createTimestampType(3), "TIMESTAMP '9999-12-31 23:59:59.999'") // max value in Exasol
+                .execute(getQueryRunner(), session, exasolCreateAndInsert(TEST_SCHEMA + "." + "test_timestamp"));
+    }
+
+    @Test
+    public void testUnsupportedType()
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = exasolServer.getSqlExecutor();
+        String table = TEST_SCHEMA + "." + "test_forced_varchar_mapping";
```

Comment:
Avoid using static table name. Also, this name is unrelated to the test purpose in my opinion.